src/engine/enginemaster.h

```diff
@@ -287,28 +307,36 @@ class EngineMaster : public QObject, public AudioSource {
     QVarLengthArray<ChannelInfo*, kPreallocatedChannels> m_activeBusChannels[3];
     QVarLengthArray<ChannelInfo*, kPreallocatedChannels> m_activeHeadphoneChannels;
     QVarLengthArray<ChannelInfo*, kPreallocatedChannels> m_activeTalkoverChannels;
+    QVarLengthArray<ChannelInfo*, kPreallocatedChannels> m_activeTalkoverHeadphoneChannels;
 
     // Mixing buffers for each output.
     CSAMPLE* m_pOutputBusBuffers[3];
+    CSAMPLE* m_pBooth;
     CSAMPLE* m_pHead;
     CSAMPLE* m_pTalkover;
-
-    CSAMPLE** m_ppSidechain; // points to master or to talkover buffer
+    CSAMPLE* m_pTalkoverHeadphones;
+    CSAMPLE* m_pSidechainMix;
+    CSAMPLE** m_ppSidechainOutput;
 
     EngineWorkerScheduler* m_pWorkerScheduler;
     EngineSync* m_pMasterSync;
 
     ControlObject* m_pMasterGain;
+    ControlObject* m_pBoothGain;
     ControlObject* m_pHeadGain;
     ControlObject* m_pMasterSampleRate;
     ControlObject* m_pMasterLatency;
     ControlObject* m_pMasterAudioBufferSize;
     ControlObject* m_pAudioLatencyOverloadCount;
+    ControlObject* m_pNumMicsConfigured;
     ControlPotmeter* m_pAudioLatencyUsage;
     ControlPotmeter* m_pAudioLatencyOverload;
     EngineTalkoverDucking* m_pTalkoverDucking;
     EngineDelay* m_pMasterDelay;
     EngineDelay* m_pHeadDelay;
+    EngineDelay* m_pBoothDelay;
+    EngineDelay* m_pInputLatencyCompensationDelay;
+    EngineDelay* m_pInputLatencyCompensationHeadphonesDelay;
```

Comment:
Please remove this. The user must not hear the delay. Or is there a use case for this?